### PR TITLE
Strip non unicode spaces from last author

### PR DESCRIPTION
**Pull request by davismr**
This should also fix: https://github.com/python-excel/xlrd/issues/288

### CODE REVIEW

File: xlrd/book.py [hunk 1]
```diff
@@ -1159,7 +1159,7 @@ def handle_sst(self, data):
         # DEBUG = 1
         if DEBUG:
             print("SST Processing", file=self.logfile)
-            t0 = time.time()
+            t0 = perf_counter()
```
Comment (jdufresne) [inline xlrd/book.py]: Is this line change unrelated?

Comment (nayyarv) [reply xlrd/book.py]: If you're going to accept #305, I already make this change in my various cleanup and would appreciate not having to resolve git conflicts.

Comment (nayyarv) [reply xlrd/book.py]: Oh wait, you've already made this change!

**Comment by davismr** [comment]
I can't anonymise the spreadsheet I have. If you have an anonymised spreadsheet with the same issue, I can add a test.

**Comment by cjw296** [comment]
CI failing -> nope.

**Pull request closed without merging**